### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by bhack** [yamy-cheng/DMAOT-VOTS2023#1]
Can we use the train in `DMAOT/tools` to retrain this on VOS as with the aot-benchmark repo?

**Comment by yamy-cheng** [comment]
Yes, the training process of DMAOT is the same as AOT. However, please note that the `dmaot_engine` is only used in the inference process and is not suitable for training. For training, you should use the `deaot_engine`.

**Comment by bhack** [comment]
So is `deaot_engine`  compatible with all the `aot-benchmaerk` pretrained model right?

Can you share some info about the model you have in the Readme?
> We also utilize the SwinB-DeAOT model trained on a larger dataset, and the pretrained weights can be downloaded from [here](https://drive.google.com/file/d/1KTJdR354BtFEGTrA_a4fcKUUHe2woxtn/view?usp=drive_link).

Can we reproduce this pretrained checkpoint?

**Comment by yamy-cheng** [comment]
Simply put, you can use the `aot-benchmark` to train the model and then use the `dmaot_engine` to test the model.  We use YouTubeVOS, Davis, MOSE, OVIS, LaSOT, and GOT-10k in the main training of SwinB-DeAOT-L to achieve better results.

**Comment by bhack** [comment]
ok thanks

**Comment by bhack** [comment]
> We use YouTubeVOS, Davis, MOSE, OVIS, LaSOT, and GOT-10k in the main training of SwinB-DeAOT-L to achieve better results.

Do you plan to release the config to reproduce this checkpoint?

**Comment by yamy-cheng** [comment]
Yes, the configuration will be released soon.

**Comment by bhack** [comment]
> Yes, the configuration will be released soon.

In the meant time we wait for the full conf/protocol to reproduce the checkpoint can you share if it was standard trained in classical AOT two stages:

- static image
- freeze encoder + BN
- train on video datasets.
